posix.mak

```diff
@@ -603,7 +604,8 @@ style_lint_shellcmds:
 	grep -nr 'import std.algorithm : ' $$(find etc std -name '*.d') ; test $$? -eq 1
 
 	@echo "Enforce Allman style"
-	grep -nrE '(if|for|foreach|foreach_reverse|while|unittest|switch|else|version) .*{$$' $$(find etc std -name '*.d'); test $$? -eq 1
+    # do not do whitespace checks on generated unicode tables
+	grep -nrE '(if|for|foreach|foreach_reverse|while|unittest|switch|else|version) .*{$$' $$(find etc std -name '*.d' -not -wholename 'std/internal/unicode_*.d'); test $$? -eq 1
```

Comment:
Ditto this (add a newline between `) {`).

Reply:
Done locally